en/docs/design/create-api/create-streaming-api/create-a-websub-streaming-api.md

```diff
@@ -0,0 +1,140 @@
+        
+        The newly added topic is displayed as follows.
+            
+        [![WebSub API Newly Added Topic]({{base_path}}/assets/img/learn/design-api/streaming-api/websub/websub-api-newly-added-topic.png)]({{base_path}}/assets/img/learn/design-api/streaming-api/websub/websub-api-newly-added-topic.png)
+
+        Expand the created topic to view its **Callback URL**. This can be used when registering the topic with a webhook provider. For a detailed tutorial, see [Create and Publish a WebSub API]({{base_path}}/tutorials/streaming-api/create-and-publish-websub-api).
+
+
+### View the AsyncAPI Definition
+
+Click **API Definition**. The Async Specification of API appears.
+
+
+### View the AsyncAPI Definition
+
+Click **API Definition**. The Async Specification of API appears.
+
+
+### Configure the Runtime Configurations
+
+1. Click **Runtime Configuration**. 
+
+    Transport Level Security  defines the transport protocol on which the API is exposed.  
+
+    [![WebSub API Runtime Configurations Page]({{base_path}}/assets/img/learn/design-api/streaming-api/websub/websub-api-runtime-configurations.png)]({{base_path}}/assets/img/learn/design-api/streaming-api/websub/websub-api-runtime-configurations.png)
+
+2. If you wish to limit the API availability to only one transport (e.g., HTTPS), uncheck the **Transport Level Security** checkbox.
+
+    Both HTTP and HTTPS transports are selected by default.
+
+### Generate a Secret
+
+When required by a webhook provider (eg: GitHub), the secret generated by a WebSub API can be used to register the WebSub API with the webhook provider. This is used by the webhook provider to sign the payload when sending events back to the WebSub API.
+     
+1. Click **Runtime** and navigate to Runtime Configurations. Expand the **Subscription** section.
+
+     [![WebSub API Runtime Configurations]({{base_path}}/assets/img/learn/tutorials/streaming-api/websub/websub-api-runtime-configurations.png)]({{base_path}}/assets/img/learn/tutorials/streaming-api/websub/websub-api-runtime-configurations.png)
+     
+2. Select **SHA1** as the **Signing Algorithm**.
+
+3. Click **Generate**, which will generate a secret.
+
+     [![WebSub API Generate Secret]({{base_path}}/assets/img/learn/tutorials/streaming-api/websub/websub-api-generate-secret.png)]({{base_path}}/assets/img/learn/tutorials/streaming-api/websub/websub-api-generate-secret.png)
+     
+4. Click **Save**. 
+
+     The generated secret can be used when registering the topic with a webhook provider. For a detailed tutorial, see [Create and Publish a WebSub API]({{base_path}}/tutorials/streaming-api/create-and-publish-websub-api).
+
+
+Now, you have successfully created and configured a Streaming API. Next, let's [Publish your API]({{base_path}}/learn/design-api/publish-api/publish-an-api).
```

Comment:
```suggestion
Now, you have successfully created and configured a Streaming API. Next, let us [Publish your API]({{base_path}}/learn/design-api/publish-api/publish-an-api).
```

Reply:
We use `let's` in REST API docs as well, so let's keep it this way :)